packages/pool/contracts/StateUtils.sol

```diff
@@ -188,6 +196,19 @@ contract StateUtils is IStateUtils {
         emit SetDaoAgent(daoAgent);
     }
 
+    /// @notice Called after deployment to set the addresses of the DAO Voting
+    /// apps
+    /// @param _votingApps Addresses of the DAO Api3Voting apps
+    function setVotingApps(address[] calldata _votingApps)
+        external
```

Comment:
Has no `require` statement to check the access rights of a `msg.sender`, most probably has to have `onlyDaoAgent()`